perception_and_utils_root/perception_and_utils/utils/image_utils.py

```diff
@@ -107,7 +107,9 @@ def check_bbox_intersection(bbox1: List[int], bbox2: List[int]) -> bool:
         return True
 
 
-def decorate_img_with_text(img: np.ndarray, frame_name: str, position: np.ndarray):
+def decorate_img_with_text(
+    img: np.ndarray, frame_name: str, position: np.ndarray
+) -> np.ndarray:
     """
     Helper method to label image with text
 
```

Comment:
Suggestion: frame_name --> text_string
Could you add a bit more information about what is position and whether text will be centered at that pixel or start from that pixel location?

Reply:
@zephirefaith thanks for pointing out, actually the description is not enough.

This function is PURELY for adding info about QR detection & position of (X,Y,Z) in the world where QR is detected wrt camera.
Frame_name is the frame in which the image was detected.

This decorated image is what we display on CV2 window for both spot & aria.


I updated the method name & description accordingly. Please take a look for further suggestions.

Reply:
Thanks for addressing this! Less confusing now :)